### ISSUE TITLE
About data MSD

### ISSUE THREAD

**Issue by wefwefWEF2** [cchen-cc/MA-SAM#7]
Are only fold_0 of split.pkl  used in the training and testing process, and the others are not used?

**Comment by cchen-cc** [comment]
Yes. We only used this split to be consistent with comparison methods.
